addons/hide-ads/userstyle.css

```diff
@@ -0,0 +1,32 @@
+.advertising {
+  text-decoration: underline;
+}
+
+.contains-advertising {
+  opacity: 0.5;
```

Comment:
Could this maybe be an issue for vision impaired users?

Reply:
they have the addon on, they know its supposed to hide potential adevertising. Plus, comments are not very vision impaired friendly anyway. I can of course make that a setting but i really dont think thats needed